Generated rows don't have a set 'r' attribute

I generated a simple XLSX file with py-excel-rs (v0.5.3) and fed it back to openpyxl (v3.1.5) and got this error:

```
self = <openpyxl.worksheet._reader.WorkSheetParser object at 0x301910e80>
row = <Element '{http://schemas.openxmlformats.org/spreadsheetml/2006/main}row' at 0x3012577e0>

    def parse_row(self, row):
        attrs = dict(row.attrib)
    
        if "r" in attrs:
            try:
>               self.row_counter = int(attrs["r"])
E               ValueError: invalid literal for int() with base 10: ''

../../.local/share/virtualenvs/myproject/lib/python3.10/site-packages/openpyxl/worksheet/_reader.py:295: ValueError
```

When I inspect the written sheet.xml, I see the r attribute is empty:

```xml
<?xml version="1.0" encoding="UTF-8" standalone="yes"?>
<worksheet xmlns="http://schemas.openxmlformats.org/spreadsheetml/2006/main" xmlns:r="http://schemas.openxmlformats.org/officeDocument/2006/relationships">
<sheetData>
<row r=""><c r="A" t="str"><v>asd</v>...</row>
</sheetData>
</worksheet>
```

I'm not familar with the XML spec for XLSX files, but I think this is for the row number? I see in the code that a row number is supposed to be there, but I'm not sure why it's blank.
https://github.com/carlvoller/excel-rs/blob/f2ca35707990625e56c570faf4646da2e6f79b21/crates/excel-rs-xlsx/src/sheet.rs#L51-L53

Notably, openpyxl doesn't strictly need it. But expects it to be a valid int/float if given.
https://foss.heptapod.net/openpyxl/openpyxl/-/blob/branch/3.1/openpyxl/worksheet/_reader.py?ref_type=heads#L285-298

That said, this file opens fine in both Excel and Google Sheets (after importing).

--- a/crates/excel-rs-xlsx/src/sheet.rs
+++ b/crates/excel-rs-xlsx/src/sheet.rs
@@ -43,6 +43,8 @@ impl<'a, W: Write + Seek> Sheet<'a, W> {
 
     // TOOD: Use ShortVec over Vec for cell ID
     pub fn write_row(&mut self, data: Vec<&[u8]>) -> Result<()> {
+        self.current_row_num += 1;
+
         let mut final_vec = Vec::with_capacity(512 * data.len());
 
         // TODO: Proper Error Handling

